app/views/contact/new.html.slim

```diff
@@ -1,28 +1,6 @@
 - title t('titles.contact')
 
 h1.h3.my0 = t('headings.contact')
```

Comment:
This can be simplified as:
```
h1.h3 = t('headings.contact')
p = t('contact.call_center')
```

Which should also add a bit of space below the heading.